hack/azure-CVM-image-create-job.yaml

```diff
@@ -0,0 +1,150 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: azure-confidential-image-creation
+  namespace: openshift-sandboxed-containers-operator
+spec:
+  parallelism: 1
+  completions: 1
+  backoffLimit: 1
+  template:
+    metadata:
+      name: azure-confidential-image-creation
+    spec:
+      volumes:
+      - name: shared-data
+        emptyDir: {}
+      - name: image-id
+        emptyDir: {}
+
+      initContainers:
+      - name: payload
+        image: quay.io/openshift_sandboxed_containers/openshift-sandboxed-containers-payload:rhel92cvm
+        imagePullPolicy: Always
+        volumeMounts:
+        - name: shared-data
+          mountPath: /payload
+        command: ["/bin/sh"]
+        args: ["-c", "cp /podvm-binaries.tar.gz /payload/"]
+
+      containers:
+      - name: create
+        image: registry.access.redhat.com/ubi9/ubi:9.2
+        securityContext:
+          runAsUser: 0 # needed for container mode dnf access
+        volumeMounts:
+        - name: shared-data
+          mountPath: /payload
+        - name: image-id
+          mountPath: /output
+        env:
+          - name: PODVM_DISTRO
+            value: rhel
+          - name: PUBLISHER
+            value: "redhat"
```

Comment:
I don't know if that matters but the equivalent job for peer pods has "RedHat".

Reply:
It's the same